tests/integration/agenda/test_viewsets.py

```diff
@@ -0,0 +1,179 @@
+from django.contrib.auth import get_user_model
+from django.core.urlresolvers import reverse
+from rest_framework.test import APIClient
+
+from openslides.agenda.models import Item, Speaker
+from openslides.utils.test import TestCase
+
+
+class ManageSpeaker(TestCase):
+    """
+    Tests managing speakers.
+    """
+    def setUp(self):
+        self.client = APIClient()
+        self.client.login(username='admin', password='admin')
+        self.item = Item.objects.create(title='test_title_aZaedij4gohn5eeQu8fe')
+        self.user = get_user_model().objects.create_user(
```

Comment:
You can use this line, but I understand our integration test, that they test the code with out User app. And that there is at least one user with the pk 1 in it (the admin user). So I will continue to import the User app directly.